app/services/doc_auth/mock/doc_auth_mock_client.rb

```diff
@@ -48,18 +46,14 @@ def post_back_image(image:, instance_id:)
         DocAuth::Response.new(success: true)
       end
 
-      def post_selfie(image:, instance_id:)
-        return mocked_response_for_method(__method__) if method_mocked?(__method__)
-
-        self.class.last_uploaded_selfie_image = image
-        DocAuth::Response.new(success: true)
+      def post_selfie
+        raise NotImplementedError,
```

Comment:
is there a spec that ensures this method is defined? if not, I would just delete it because we still get a `NoMethodError` when calling it, almost the same as `NotImplementedError`

Reply:
Yup [there is](https://github.com/18F/identity-idp/blob/a1271473dd804bba7ef7da33c180ae697d1e80f2/spec/services/doc_auth/mock/dock_auth_mock_client_spec.rb#L6)